templates/s3_cors.json.j2

```diff
@@ -1,9 +1,10 @@
 {
   "CORSRules": [
      {
-        "AllowedOrigins": ["{{ siteUrl }}"],
         "AllowedHeaders": ["*"],
-        "AllowedMethods": ["PUT", "GET"]
+        "ExposedHeaders": ["ETag"],
```

Comment:
This is ExposeHeader

Reply:
whoop! Leonid used "ExposeHeaders" - AWS doc uses singular.

Reply:
Wait, I'm confused now - it looks like the AWS  doc actually uses plural too ("ExposeHeaders"): https://docs.aws.amazon.com/cli/latest/reference/s3api/put-bucket-cors.html

Reply:
Sorry - the issue was the past -tense of Exposed vs Expose - didn't mean to also drop the 's'.